community/ibm-rethinkdb-dev/README.md

```diff
@@ -0,0 +1,101 @@
+# RethinkDB
+
+[RethinkDB](https://github.com/rethinkdb/rethinkdb) Open-source database for building realtime web applications.
+
+```console
+$ helm install community/ibm-rethinkdb-dev
+```
+
+## Prerequisites
+
+- Kubernetes 1.7+ 
+- Tiller 2.7.2 or later
+
+## Resources Required
+The chart deploys pods consuming minimum resources.
+
+## PodSecurityPolicy Requirements
+This chart requires a PodSecurityPolicy to be bound to the target namespace prior to installation. Choose predefined ibm-anyuid-psp PodSecurityPolicy.
+
+## Introduction
+
+[RethinkDB](https://github.com/rethinkdb/rethinkdb) Open-source database for building realtime web applications.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm intall --name my-release community/ibm-rethinkdb-dev
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Chart Details
+This chart bootstraps a [RethinkDB](https://github.com/rethinkdb/rethinkdb) deployment on a [Kubernetes](http://kubernetes.io) cluster
+
+
+## Configuration
+
+The following table lists the configurable parameters of the rethinkdb chart and their default values.
+
+|      Parameter            |          Description            |                         Default                         |
+|---------------------------|---------------------------------|---------------------------------------------------------|
+| `image`                   | The image to pull and run       | `ibmcom/rethinkdb-ppc64le:2.3.6`                        |
+| `imagePullPolicy`         | Image pull policy               | `Always` if `imageTag` is `latest`, else `IfNotPresent` |
+| `node`                    | Specify what architecture Node  | `ppc64le`                                               |
```

Comment:
Sorry, I missed this one from before; we should add the rest of the charts configurable values to this table.